Revise `RowMetaData#getColumnMetadata(String)`

jchrys opened the pull request:
Motivation:
The current behavior does not align with the specification

Modifications:
implement the intended behavior as per the specification.

Result:
Correct behavior

jchrys committed:
Revise `RowMetaData#getColumnMetadata(String)`

Motivation:
The current behavior does not align with the specification

Modifications:
implement the intended behavior as per the specification.

Result:
Correct behavior

mirromutth commented on r2dbc-mysql/src/test/java/io/asyncer/r2dbc/mysql/MySqlRowDescriptorTest.java:
```diff
@@ -0,0 +1,65 @@
+    }
+
+
+}
```
Maybe a line break?

jchrys replied:
Sure, Let me add it.
(check style should've thrown an error. let me check)

jchrys replied:
Currently, the Checkstyle plugin does not analyze source code located under the test directory. For the time being, I prefer to keep it.

mirromutth reviewed (approved):
LTGM, only a code style nit

jchrys committed:
Revise `RowMetaData#getColumnMetadata(String)`

Motivation:
The current behavior does not align with the specification

Modifications:
implement the intended behavior as per the specification.

Result:
Correct behavior
